runtime/src/serde_snapshot.rs

```diff
@@ -1040,14 +1043,10 @@ where
         exit,
     );
 
-    let AccountsDbFields(
-        _snapshot_storages,
-        snapshot_version,
-        _snapshot_slot,
-        snapshot_bank_hash_info,
-        _snapshot_historical_roots,
-        _snapshot_historical_roots_with_hash,
-    ) = snapshot_accounts_db_fields.collapse_into()?;
+    snapshot_accounts_db_fields.verify_slot_ranges()?;
+
+    let (snapshot_version, snapshot_bank_hash_info) =
```

Comment:
I found somewhere that `snapshot_version` is obsolete, maybe we don't need it. It is only used below to increment some atomic, the same atomic is used in other places to add... lengths: https://github.com/anza-xyz/agave/blob/bcaeadd992fbb8daab77801af96c962f24791fd0/accounts-db/src/accounts_db.rs#L5968